src/Model/Behavior/RegisterBehavior.php

```diff
@@ -31,11 +31,11 @@ class RegisterBehavior extends BaseTokenBehavior
     /**
      * @var bool
      */
-    protected $validateEmail;
+    protected bool $validateEmail = true;
     /**
      * @var bool
      */
-    protected $useTos;
+    protected bool $useTos = true;
```

Comment:
The default value would be the one from config.

We have  `$this->useTos = (bool)Configure::read('Users.Tos.required');`

And since the default value for the config `Users.Tos.required`  is true it make sense to set true here too.

Reply:
OK let's add it to the migration guide in case other could be extending the behavior